src/components/cylc/workflow/index.js

```diff
@@ -16,223 +16,9 @@
-    document.getElementById(this.widgetId).addEventListener('delete:widgetcomponent', this.delete)
-    document.getElementById(this.widgetId).addEventListener('activate:widgetcomponent', this.activate)
-  },
-  beforeDestroy () {
-    document.getElementById(this.widgetId).removeEventListener('delete:widgetcomponent', this.delete)
-    document.getElementById(this.widgetId).removeEventListener('activate:widgetcomponent', this.activate)
-  },
-  methods: {
-    delete () {
-      // This is captured by the View, that holds subscriptions, and then used to tell which subscription must be turned off
-      EventBus.$emit('delete:widget', { id: this.widgetId })
-      this.$destroy()
-    },
-    activate () {
-      // when this widget is activated, we want to tell the wrapped graph that it needs to force-repaint to avoid blank graphs
-      const children = this.$refs[this.widgetId].$children
-      if (children && children.length > 0) {
-        children[0].resizeGraph()
-      }
-    }
-  },
-  template: `
-    <div>
-      <v-skeleton-loader
-        :ref="widgetId"
-        :loading="isLoading"
-        height="100"
-        type="list-item-avatar-three-line"
-      >
-        <graph :workflow-name="workflowName" />
-      </v-skeleton-loader>
-    </div>
-  `
-})
-
-const MutationsWrapper = Vue.component('mutations-wrapper', {
-  name: 'MutationsWrapper',
-  props: {
-    widgetId: {
-      type: String,
-      required: true
-    },
-    workflowName: {
-      type: String,
-      required: true
-    },
-    isLoading: {
-      type: Boolean,
-      default: false
-    }
-  },
-  components: {
-    Mutations,
-    VSkeletonLoader
-  },
-  methods: {
-    delete () {
-      // This is captured by the View, that holds subscriptions, and then used to tell which subscription must be turned off
-      EventBus.$emit('delete:widget', { id: this.widgetId })
-      this.$destroy()
-    }
-  },
-  mounted () {
-    const widgetElement = document.getElementById(this.widgetId)
-    widgetElement.appendChild(this.$refs[this.widgetId].$el)
-    const vm = this
-    document.getElementById(this.widgetId).addEventListener('delete:widgetcomponent', () => {
-      EventBus.$emit('delete:widget', { id: vm.widgetId })
-      vm.$destroy()
-    }, false)
-  },
-  beforeDestroy () {
-    document.getElementById(this.widgetId).removeEventListener('delete:widgetcomponent', this.delete)
-    document.getElementById(this.widgetId).removeEventListener('activate:widgetcomponent', this.activate)
-  },
-  template: `
-    <div>
-      <component
-        is="VSkeletonLoader"
-        :ref="widgetId"
-        :loading="isLoading"
-        height="100"
-        type="list-item-avatar-three-line"
-      >
-        <Mutations :workflow-name="workflowName" />
-      </component>
-    </div>
-  `
-})
 
 const EventBus = new Vue()
 
 export {
-  ContentWidget, TreeWrapper, GraphWrapper, MutationsWrapper, EventBus
+  EventBus
 }
```

Comment:
For Vue 3 we must get rid of the event bus. But that's for another pull request.